packages/main/src/ResponsivePopover.hbs

```diff
@@ -9,7 +9,19 @@
 		@ui5-afterClose="{{_afterDialogClose}}"
 	>
 		{{!-- forward slot header to inner dialog slot header --}}
-		<slot slot="header" name="header"></slot>
+		{{#if header.length}}
+			<slot slot="header" name="header"></slot>
+		{{else}}
+			<header class="ui5-responsive-popover-header">
```

Comment:
probably the headerText should be also displayed if set

Reply:
Done